app/controllers/enlistments_controller.rb

```diff
@@ -32,7 +33,7 @@ def create
       redirect_to project_enlistments_path(@project)
     else
       flash[:error] = @code_location.errors['error']
-      render :new, status: :unprocessable_entity
```

Comment:
Why does the status need to be changed?

Reply:
I just checked other places like review_controller, We dont use this status as we render new link on unauthorization.

So, just to maintain the code consistency , I have also removed the status code.